### ISSUE TITLE
Add ignoreSelectors: [] to selector-no-vendor-prefix

### ISSUE THREAD

**Issue by sindresorhus** [stylelint/stylelint#3197]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I use Stylelint with an Electron app where I have to use a few vendor prefixed selectors.

Same as https://github.com/stylelint/stylelint/issues/3007

> Which rule, if any, is this issue related to?

`selector-no-vendor-prefix`

> What CSS is needed to reproduce this issue?

```css
input::-webkit-input-placeholder {
  color: pink;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "selector-no-vendor-prefix": true
  }
}
```

> Which version of stylelint are you using?

9.1.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

I expected these rules to have an ignore option like many other rules have.

> What actually happened (e.g. what warnings or errors you are getting)?

The vendor prefixes got reported.

**Comment by evilebottnawi** [comment]
@sindresorhus Thanks for issue, PR welcome :+1:

**Comment by evilebottnawi** [comment]
Anyway i think we should ignore by default all non prefixable selectors

**Comment by sindresorhus** [comment]
> Anyway i think we should ignore by default all non prefixable selectors

I agree. Same with to `property-no-vendor-prefix` and `value-no-vendor-prefix`.
